contrib/apply_copyright.rb

```diff
@@ -0,0 +1,41 @@
+#! /usr/bin/env ruby
+
+$copyright = <<EOF
+/*
+ * FILENAME
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2018 Apple Inc. and the FoundationDB project authors
```

Comment:
```suggestion
 * Copyright 2013-2021 Apple Inc. and the FoundationDB project authors
```

You need to change copyright year for files modified to 2021.

Reply:
This was not changed to 2021.